pages/getting-started/install-memgraph/docker.mdx

```diff
@@ -227,14 +63,14 @@ terminal:
 docker exec -it memgraph mgconsole
 ```
 
-You can also connect to Memgraph instance using the [Memgraph Platform desktop
-application](/data-visualization/user-manual), available for download from
-[Memgraph's Download Hub](https://memgraph.com/download). 
+You can also connect to Memgraph instance using the [Memgraph
+Lab](/data-visualization/user-manual), available for download from [Memgraph's
+Download Hub](https://memgraph.com/download). 
```

Comment:
Or using the Memgraph Lab Docker image (link to below section that is currently missing)

Reply:
Link aded. Still working on the content.